workflow/executor/executor.go

```diff
@@ -39,6 +39,13 @@ import (
 	os_specific "github.com/argoproj/argo/workflow/executor/os-specific"
 )
 
+var MainContainerStartRetry = wait.Backoff{
+	Steps:    8,
+	Duration: 1 * time.Second,
+	Factor:   1.0,
+	Jitter:   0.1,
+}
```

Comment:
How did we decide that this was the best backoff?

Reply:
We decide these values after tests on our cluster, at first we wanted to use DefaultRetry which was too low and not configuration.